fluentd/generate_throttle_configs.rb

```diff
@@ -108,6 +109,7 @@ def create_default_docker(excluded)
   @label @INGRESS
   path /var/log/containers/*.log
   pos_file /var/log/es-containers.log.pos
+  encoding UTF-8
```

Comment:
We have the same utf-8 problem with container log files as well as /var/log/messages?

Reply:
If they log UTF-8 messages, then yes.